### ISSUE TITLE
I have a problem with exporting animation with masks in json

### ISSUE THREAD

**Issue by Neesrana** [airbnb/lottie-web#2728]
Hi!
I have a problem with exporting animations with masks to Bodymovin or Lottifiles.
Preview in aftereffects looks good, but when I open json, it all breaks all masks.
Maybe I have to make some tricky actions with masks?

After Effects file attached
[anime_logo.zip](https://github.com/airbnb/lottie-web/files/8161908/anime_logo.zip)
.

**Comment by bodymovin** [comment]
hi, can you share the project with its related files?

**Comment by Neesrana** [comment]
Yes sorry i forgot to create a file without related form, in this file you can find my corrected after effect animation and my logotype in a illustrator file.
Thanks for your help !
[logo_neesrana.zip](https://github.com/airbnb/lottie-web/files/8349919/logo_neesrana.zip)
